recipes/oslo.serialization/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{%set name = "oslo.serialization" %}
+{%set version = "2.13.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "7bfdb1c10bc24c4887407a2ac46708b1fd16e9bc2ceb6bb3805696e8cebf6566" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.6
+    - six >=1.9.0
+    - msgpack-python >=0.4.0
```

Comment:
Do we have a PR here at staged-recipes for this one? If not, could we please add one?